.github/workflows/build-windows.yml

```diff
@@ -4,18 +4,29 @@ on:
   push:
     tags:
       - "**"
-# on: [push]
+
 jobs:
   Build-Windows:
     runs-on: windows-latest
     steps:
-      # Use Python 3.12.4
       - uses: actions/checkout@v4
+      
       - uses: actions/setup-python@v5
         with:
           python-version: "3.12.4"
-      - run: pip install -r requirements.txt && pip install pyinstaller && python3 helper_scripts/package_windows.py
-      - uses: actions/upload-artifact@v4
+      
+      - name: Install dependencies and build
+        run: |
+          pip install -r requirements.txt
+          pip install pyinstaller
+          python helper_scripts/package_windows.py
+      
+      - name: Rename and Compress Executable
+        run: |
+          Move-Item -Path ".\dist\Qtcord-Windows\Qtcord.exe" -Destination ".\Qtcord-${{github.ref_name}}-Windows-x86_64.exe"
```

Comment:
![image](https://github.com/user-attachments/assets/a42e1478-b6e7-4adf-9dbc-3d68f412dee8)

Reply:
Did you forget to include everything in the folder `Qtcord-Windows`?